script/Deploy.s.sol

```diff
@@ -1,16 +1,12 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: MIT
```

Comment:
There's no executable script in this file, is the plan to delete it or to modify it?

Reply:
I think scopelint requires this and we plan to switch over.

Reply:
wdym? scopelint doesn't require a script to be present, so we can delete the file if we don't use it

Reply:
Sorry, what I was referring to is that scopelint requires a script directory. If we delete the file the directory is also [removed](https://github.com/ScopeLift/flexible-voting-clients/tree/261c0a7b3f60bce9c3d74124a1b69ca7428f982d). We could add an empty git.keep file or something so the directory stays.

Reply:
https://github.com/ScopeLift/flexible-voting-clients/pull/1/commits/2ae269aa3543c1027e63b1ce17ac9332e4b7addb

Reply:
Ah I see, yea `.gitkeep` makes sense, resolving